infra/modules/providers/azure/api-mgmt/README.md

```diff
@@ -0,0 +1,51 @@
+# Module Azure API Management
+
+Azure API Management is a turnkey solution for publishing APIs to external and internal customers. It quickly creates consistent and modern API gateways for existing back-end services hosted anywhere.
+
+More information for Azure API Management Service can be found [here](https://azure.microsoft.com/en-us/services/api-management)
+
+A terraform module in Cobalt to provide API manangement with the following characteristics:
+
+- Ability to specify resource group name in which the API manager is deployed.
+- Ability to specify resource group location in which the API manager is deployed.
+- Also gives ability to specify the following for API Manager based on the requirements:
+  - name : The name of the API Manager to be deployed.
+  - publisher_name : The name of the Publisher/Company of the API Management Service.
+  - publisher_email : The email of Publisher/Company of the API Management Service.
+  - tags : A mapping of tags assigned to the resource.
+  - sku_name : Specifies the plan's pricing tier.
+  - capacity : Specifies the number of units associated with this API Management service.
+
+Please click the [link](https://www.terraform.io/docs/providers/azurerm/d/api_management.html) to get additional details on settings in Terraform for Azure API Management.
+
+## Usage
```

Comment:
Usage should represent module usage as opposed to the the standard azure resource capabilities.